src/marten/db/query/prefetcher.cr

```diff
@@ -166,20 +171,32 @@ module Marten
           records_to_decorate
         end
 
+        private def prefetch_query_set(relation_name, default_model)
```

Comment:
Can we rename this method to something like `#query_set_for_prefetched_relation`? 🙏